R/pkg/R/context.R

```diff
@@ -225,3 +225,17 @@ broadcast <- function(sc, object) {
 setCheckpointDir <- function(sc, dirName) {
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(dirName))))
 }
+
+#' Set new log level
+#'
+#' Set new log level: "ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE", "WARN"
+#' @param sc existing spark context
```

Comment:
"existing spark context" reads a bit oddly, maybe go with "Spark Context to use" which is what the sc param is called in the other functions in this file for consistencies sake?